style.css

```diff
@@ -1,3 +1,72 @@
+@import url('https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Roboto+Slab:wght@100;200;300;400;500;600;700;800;900&display=swap');
+
 * {
   box-sizing: border-box;
```

Comment:
- [ ]  Kyndly stick to the figma design as much as possible

- Please use the correct font size 

![image](https://user-images.githubusercontent.com/95243996/191749996-b6b661c0-4eec-4fa6-8e7b-3d916e1e2bd0.png)

Reply:
@andgarzonmal, I've made all the changes you instructed. Can you please check it from here before I submit it from the dashboard?? I have no extra reviews. Please......